fe/fe-core/src/main/java/com/starrocks/persist/EditLog.java

```diff
@@ -1400,6 +1401,10 @@ public void logModifyEnablePersistentIndex(ModifyTablePropertyOperationLog info)
         logEdit(OperationType.OP_MODIFY_ENABLE_PERSISTENT_INDEX, info);
     }
 
+    public void logModifyWriteQuorum(ModifyTablePropertyOperationLog info) {
+        logEdit(OperationType.OP_MODIFY_WRITE_QUORUM, info);
```

Comment:
Does it throw a compatibility problem? @gengjun-git

Reply:
No, the lower version view ignore this log